javascript/src/main/components/BasicCourseSearch/BasicCourseTable.js

```diff
@@ -100,6 +103,19 @@ const BasicCourseTable = ({ classes, checks, displayQuarter, allowExport }) => {
     }
   }
 
+  
+  const { getAccessTokenSilently: getToken } = useAuth0();
+  const { data: roleInfo } = useSWR(
+    ["/api/myRole", getToken],
+    fetchWithToken
+  );
+  const isAdmin = roleInfo && roleInfo.role.toLowerCase() === "admin";
+  const isMember = roleInfo && roleInfo.role.toLowerCase() === "member";
+  const isGuest = roleInfo && roleInfo.role.toLowerCase() === "guest";
+  const isLogedIn = isAdmin || isMember || isGuest
```

Comment:
should change spelling of const var 'isLogedIn' to 'isLoggedIn'

Reply:
Got it thanks so much for catching that !